Library/Formula/vim.rb

```diff
@@ -2,8 +2,8 @@ class Vim < Formula
   homepage "http://www.vim.org/"
   head "https://vim.googlecode.com/hg/"
   # This package tracks debian-unstable: http://packages.debian.org/unstable/vim
-  url "http://ftp.debian.org/debian/pool/main/v/vim/vim_7.4.488.orig.tar.gz"
-  sha256 "5b58ef7d4ce2c2eb84af2f3dbb3b5d6919adc97f12de1509a13681fa58936faf"
+  url "http://ftp.debian.org/debian/pool/main/v/vim/vim_7.4.712.orig.tar.gz"
```

Comment:
Can we use ` https://mirrors.kernel.org/debian/pool/main/v/vim/vim_7.4.712.orig.tar.gz ` here instead?